sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -55,40 +56,40 @@ and then include the direct dependency in the dependencies section without the v
 If you want to take dependency on a particular version of the library that is not present in the BOM,
 add the direct dependency to your project as follows.
 
-[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;dependency})
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-ai-textanalytics</artifactId>
-    <version>5.1.9</version>
+    <version>5.2.0-beta.3</version>
 </dependency>
 ```
 [//]: # ({x-version-update-end})
-**Note:** This version of the client library defaults to the `v3.2-preview.1` version of the service.
+**Note:** This version of the client library defaults to the `2022-04-01-preview` version of the service.
 
 This table shows the relationship between SDK services and supported API versions of the service:
 
 |SDK version|Supported API version of service
 |-|-
-|5.2.x | 3.0, 3.1, 3.2-preview.1 (default)
+|5.2.x | 3.0, 3.1, 2022-04-01-preview (default)
```

Comment:
Consider adding a note to your changelog about dropping support for `3.2-preview.2`

Reply:
Ahh I was looking at main. thank you